modules/OCPP201/OCPP201.cpp

```diff
@@ -407,6 +407,51 @@ ocpp::v201::BootReasonEnum get_boot_reason(types::system::BootReason reason) {
     }
 }
 
+ocpp::v201::ReasonEnum get_reason(types::evse_manager::StopTransactionReason reason) {
+    switch (reason) {
+    case types::evse_manager::StopTransactionReason::DeAuthorized:
+        return ocpp::v201::ReasonEnum::DeAuthorized;
```

Comment:
For these enum convert functions, it saves some space and typing, if you're using `using` like
```
using EvseReason = types::evse_manager::StopTransactionReason;
using OcppReason = ocpp::v201::ReasonEnum;
```
inside the function. Then you can write:
```
case EvseReason::EmergencyStop:
    return OcppReason::EmergencyStop;
```
Furthermore, I would call the function `convert_reason`